tests/RelationTest.php

```diff
@@ -306,8 +307,7 @@ public function testOneToOne(): void
     }
 
     /**
-     * This test verifies the temporary behaviour of audited entities with M-M relationships
-     * until https://github.com/simplethings/EntityAudit/issues/85 is implemented.
+     * This test verifies the temporary behaviour of audited entities with M-M relationships.
```

Comment:
I think this comment is outdated. The behavior implemented right now is not temporary right?